### PR TITLE
added inputs and image components

### PR DESCRIPTION
**Pull request by Shivani-1524**
1. Inputs with validation
2. responsive images and rounded images

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  display: block;
+  font-weight: 700;
+}
+
+.user-input {
+  background-color: var(--extra-light-grey);
+  margin-top: 7px;
+  width: 100%;
+  min-height: 40px;
+  width: 100%;
```
Comment (gauravsinhaweb) [inline Components/Input/input.css]: width is repeating.

File: Components/Image/image.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.custom-image-container {
+  width: 100%;
```
Comment (gauravsinhaweb) [inline Components/Image/image.css]: use shorter classname for short typing

File: Components/Input/input.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
+textarea:focus,
+input:focus {
+  outline: none;
+}
+
+.center-block.inp-ex {
+  margin-top: 30px;
+  width: 400px;
```
Comment (gauravsinhaweb) [inline Components/Input/input.css]: using rems would give responsive width

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: documentation.html [hunk 4]
```diff
@@ -361,11 +361,100 @@ <h2 class="rg-booky-title">Text and Icon Badge</h2>
...
+                    <div class="mg-t-40">
+                        <h2 class="rg-booky-title">Example of Inputs</h2>
+                        <div class="center-block inp-ex">
+                            <div>
+                                <label class="input-label sm-title" for="name">Name</label>
+                                <input class="user-input" id="name" type="text" />
+                            </div>
+                            <div class="mg-t-10">
+                                <label class="input-label sm-title" for="name">Password</label>
+                                <input class="user-input" id="name" type="password" placeholder="6+ characters" />
```
Comment (Naman-Saxena1) [inline documentation.html]: You have used same id="name" in password and in its label, can change it to password. Might create confusion once codebase increases.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: Components/Image/image.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  height: 300px;
```
Comment (omkarpatke) [inline Components/Image/image.css]: use rem here as well

File: Components/Image/image.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+img.img-resp {
```
Comment (omkarpatke) [inline Components/Image/image.css]: this is confusing, not readable you can try an only class here.

File: Components/Input/input.css (hunk 1 above)
Comment (Shivani-1524) [reply Components/Input/input.css]: thanks for pointing it out!!

File: Components/Input/input.css (hunk 3 above)
Comment (Shivani-1524) [reply Components/Input/input.css]: agreed! changed it to % instead :)

File: Components/Image/image.css (hunk 2 above)
Comment (Shivani-1524) [reply Components/Image/image.css]: will replace image with img cant think of anything else xD

File: documentation.html (hunk 4 above)
Comment (Shivani-1524) [reply documentation.html]: i forgot to change it while copy pasting 😂 thanks for pointing it out !

File: Components/Image/image.css (hunk 6 above)
Comment (Shivani-1524) [reply Components/Image/image.css]: not sure how its confusing but i will remove the img tag to reduce specificity! thanks :)

File: Components/Image/image.css (hunk 5 above)
Comment (Shivani-1524) [reply Components/Image/image.css]: didnt want it to scale but changed units anyway :) thanks

**Pull request merged**